src/main/scala/org/broadinstitute/dsde/firecloud/service/DataUseRestrictionSupport.scala

```diff
@@ -154,6 +202,7 @@ trait DataUseRestrictionSupport extends LazyLogging {
       case (attr: AttributeName, AttributeBoolean(true)) => attr.name
     }.toSeq
 
+    //this isn't actually calling ontologyDAO?
     val dsLabels:Seq[String] = (workspace.attributes.get(diseaseLabelsAttributeName) collect {
       case value: AttributeValueList => value.list.collect {
         case a: AttributeString => "DS:" + a.value
```

Comment:
`generateUseRestrictionDisplayAttribute` is supposed to get the data use attributes out of the attributes of the workspace and then package into something like `Map("NPU", "HMB", "RS-G", "DS:some cancer")`. However, this part which appends the disease to "DS:" doesn't call an ontologyDAO, meaning the attributes already have the disease them in them? This is a bit different from the way our DS attributes are populated for the display field (we're calling ontologyDAO to get the disease names). Which makes me wonder if we should keep these two sections separate instead of refactoring them together?

Reply:
When this method is called, the disease name attributes have already been pulled from ontology otherwise they wouldn't exist. IIRC, all data-use specific attributes would have been populated during the publishing process so this method would only be looking at attributes that exist post-publish.